flamedisx/xenon/x1t_sr1.py

```diff
@@ -11,13 +10,7 @@
 
 o = tf.newaxis
 
-##
-# Yield maps
-##
-s1_map, s2_map = [
-    fd.InterpolatingMap(fd.get_resource(fd.pax_file(x)))
-    for x in ('XENON1T_s1_xyz_ly_kr83m-SR1_pax-664_fdc-adcorrtpf.json',
-              'XENON1T_s2_xy_ly_SR1_v2.2.json')]
+import pdb as pdb
```

Comment:
Probably left in from debugging. You can also set breakpoints with the built-in `breakpoint()`; if that's all you need you don't need to import pdb.

Reply:
Thanks for catching. I've removed it. Nice, I'll try out breakpoint(). I just want something similar to Matlab's debugging options (breakpoints, conditional breakpoints, stepping into functions, etc).